### PR TITLE
set account tree first default hash to be 0x290...

### PR DESCRIPTION
**Pull request by ChihChengLiang**
### What's wrong

Currently, we have different default hashes for the state tree and the public key tree. This might confuse client developers and application developers

- first default hash for state tree: `0x290decd9548b62a8d60345a988386fc84ba6bc95484008f6362f93160ef3e563`
- first default hash for public key tree: `0x0000000000000000000000000000000000000000000000000000000000000000`

### How are we fixing it

Make them the same. Set to `0x290decd9548b62a8d60345a988386fc84ba6bc95484008f6362f93160ef3e563`

### CODE REVIEW

**Review by vaibhavchellani — APPROVED** [review]
Lgtm

File: contracts/AccountTree.sol [hunk 1]
```diff
@@ -25,6 +25,13 @@ contract AccountTree {
     bytes32[DEPTH - BATCH_DEPTH] public filledSubtreesRight;
 
     constructor() public {
+
```
Comment (vaibhavchellani) [inline contracts/AccountTree.sol:28]: Can we lint on this?

**Review by vaibhavchellani — APPROVED** [review]
LGTM!

File: contracts/AccountTree.sol (hunk 1 above)
Comment (ChihChengLiang) [reply contracts/AccountTree.sol]: sure thing 👍

**Pull request merged**